# Análisis Exploratorio de los Datos

Se realiza a continuación el análisis de la data con la cual  se realiza el fine-tuning del modelo preentrenado. 

In [2]:
from google.colab import drive
drive.mount('/content/drive')
path= "/content/drive/MyDrive/MLDS-2/MODULO II/Talleres/SMSSpamCollection"

Mounted at /content/drive


In [3]:
from numpy.core.fromnumeric import shape
import pandas as pd
df=messages = pd.read_csv(path, sep='\t',
                           names=["label", "message"])
X=list(df['message'])
y=list(df['label'])
#y=list(pd.get_dummies(y,drop_first=True)['spam'])
shape(X)



(5572,)

In [4]:
shape(y)

(5572,)

La muestra con la que se realiza el fine-tuning del modelo escogido está compuesta por 5572 mensajes y sus correspondientes etiquetas. 

In [10]:
salida =pd.unique(y)
salida

array(['ham', 'spam'], dtype=object)

Se tiene que el vector de etiquetas es una lista binaria. Se encarga de etiquetar mensajes en **Spam** o no (**Ham**). 